### DISCUSSION TITLE
React  Q/A ? || #FAQs

### DISCUSSION THREAD

**Discussion by su-network** [fief-dev/fief#107] [Q&A] [answered]
> I follow the code that is written in  documentation but the problem is even after having right redirect _uri and credentials I still got error something called redirect URI is not found.

**Output**

Redirect URI in my Fief-Client : 
![image](https://user-images.githubusercontent.com/75021117/199981715-bff23511-b452-4bc6-b3b2-f96fd1b69c29.png)

After pressing login button present in navbar : 
![image](https://user-images.githubusercontent.com/75021117/199982265-d9f9083c-3668-41fa-a5c3-434bda1f8f1f.png)

Solution:

If you running the React-Vite project which is basically running on default port 5173 you have to change to port 3000 to make it work.

**Comment by frankie567** [comment]
The redirect URI must *exactly* match the one shown in your browser: this includes the scheme (http, https), the host (localhost, mydomain.com...), the port (nothing, 8000, 5173...) and the path (`/callback`, `/redirect`...).

**Comment by su-network** [comment]
@frankie567   is there any delete button to deleted unused workspace ?

**Comment by su-network** [comment]
![image](https://user-images.githubusercontent.com/75021117/202088040-b8abd322-3bff-443b-80a7-6113de821e49.png)

Hello @frankie567, the discussion is about google oauth providers,I have subdomain like hellofief.fief.dev but it's showing it need top private domain at oauth consent screen but as per as documentation we need to input our workspace subdomain like hellofief.fief.dev  it doesn't working.Is there way we can include such subdomain in our authorized domain?

**Reply by frankie567** [reply, marked as answer]
Thank you for raising this, @su-shubham! After checking, it's indeed a mistake in the documentation. The value to input here will be the top-domain `fief.dev` (actually, it seems Google add it automatically when you create the OAuth client in the next step, so you could even let it blank). I'll fix the docs!

PS: For new questions, I suggest you create a new thread; it's easier to track and find 😉
